lib/IssueBodyChecker.js

```diff
@@ -0,0 +1,81 @@
+function fromBase64 (content) {
+  return Buffer.from(content, 'base64').toString()
+}
+
+async function getIssueTemplateBody (context, path) {
+  try {
+    const singleTemplate = await context.github.repos.getContent({
+      owner: context.payload.repository.owner.login,
+      repo: context.payload.repository.name,
```

Comment:
Can we clean this up using some [probot internal APIs](https://probot.github.io/api/latest/classes/context.html#repo):

```js 
const singleTemplate = await context.github.repos.getContent(context.repo({path}))
```